tests/microbenchmarks.py

```diff
@@ -138,6 +144,7 @@ def yarn(n):
             main(i)
     """), timed=True)
 
+
 def test_cmp(test_case):
     print("Running", "test_cmp")
     test_case.runAsJava(adjust("""
```

Comment:
At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils